src/js/tech/html5.js

```diff
@@ -70,6 +72,13 @@ class Html5 extends Tech {
       }
     }
 
+    if (this.featuresNativeVideoTracks) {
+      this.handleVideoTrackChange_ = Fn.bind(this, this.handleVideoTrackChange);
```

Comment:
Vendor like safari, unfortunately, I don't remember the exact details. Was merged as part of https://github.com/videojs/video.js/pull/2425.